src/ui/widgets/Readback/readback.module.css

```diff
@@ -5,11 +5,11 @@
   white-space: nowrap;
   text-overflow: ellipsis;
   background-color: #383838;
```

Comment:
The background colour for the readback widget is currently very dark, whereas the default Phoebus background is a lighter grey - I believe the `background-color` should be `#f0f0f0`.
Current: 
![current_readback](https://github.com/dls-controls/cs-web-lib/assets/45264327/ae1d4da8-132a-4f35-b5aa-234ce894e9cd)
Proposed:
![readback_new](https://github.com/dls-controls/cs-web-lib/assets/45264327/85b74804-87f1-4916-8fbf-01a7723c8923)

Reply:
Thanks for the spot, changed this now.